google_analytics/tests/test_ga.py

```diff
@@ -276,6 +276,33 @@ def test_build_ga_params_for_custom_params(self):
         self.assertEqual(
             parse_qs(ga_dict_with_custom.get('utm_url')).get('key'), ['value'])
 
+    @responses.activate
+    def test_build_ga_params_for_campaign_tracking_params(self):
+        '''
+        Test that the  correct GA campaign
+        tracking params are tracked correctly
+        '''
+        request = self.make_fake_request(
+            '/somewhere/?cn=campaign name&cs=campaign source')
```

Comment:
The parameters in this url need to be the `utm` ones.
The parameters in the inbound url will still be the same (`utm_medium` ect) but when we send it to GoogleAnalytics we need to map the old parameters to the ones that GA expects (`cn`, `cs`, etc)

Reply:
oooh, thank you